Alloy/template/lib/alloy.js

```diff
@@ -32,9 +32,8 @@ exports._ = _;
 exports.Backbone = Backbone;
 
 var DEFAULT_WIDGET = 'widget';
-var TI_VERSION = Ti.version;
-var MW320_CHECK = OS_MOBILEWEB && TI_VERSION >= '3.2.0';
-var IDENTITY_TRANSFORM = OS_ANDROID ? (Ti.version >= '8.0.0' ? Ti.UI.createMatrix2D() : Ti.UI.create2DMatrix()) : undefined;
+var MW320_CHECK = OS_MOBILEWEB;
+var IDENTITY_TRANSFORM = OS_ANDROID ? (Ti.UI.createMatrix3D ? Ti.UI.createMatrix2D() : Ti.UI.create2DMatrix()) : undefined;
```

Comment:
You should be checking for `Ti.UI.createMatrix2D` existence.
I think this is a copy-and-paste mistake. :stuck_out_tongue: